<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-04-20T10:16:26 IST" name="Suite" finished-at="2024-04-20T10:17:02 IST" duration-ms="36370">
    <groups>
      <group name="regression">
        <method signature="Tc_001_AccountRegistrationTest.verify_account_registration()[pri:0, instance:testCases.Tc_001_AccountRegistrationTest@7714e963]" name="verify_account_registration" class="testCases.Tc_001_AccountRegistrationTest"/>
      </group> <!-- regression -->
      <group name="master">
        <method signature="Tc_001_AccountRegistrationTest.verify_account_registration()[pri:0, instance:testCases.Tc_001_AccountRegistrationTest@7714e963]" name="verify_account_registration" class="testCases.Tc_001_AccountRegistrationTest"/>
      </group> <!-- master -->
    </groups>
    <test started-at="2024-04-20T10:16:26 IST" name="Test" finished-at="2024-04-20T10:17:02 IST" duration-ms="36370">
      <class name="testCases.Tc_001_AccountRegistrationTest">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.Tc_001_AccountRegistrationTest@7714e963]" started-at="2024-04-20T10:16:26 IST" name="setup" finished-at="2024-04-20T10:16:54 IST" duration-ms="27770" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[edge]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verify_account_registration()[pri:0, instance:testCases.Tc_001_AccountRegistrationTest@7714e963]" started-at="2024-04-20T10:16:54 IST" name="verify_account_registration" finished-at="2024-04-20T10:17:00 IST" duration-ms="6415" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at testCases.Tc_001_AccountRegistrationTest.verify_account_registration(Tc_001_AccountRegistrationTest.java:63)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_account_registration -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.Tc_001_AccountRegistrationTest@7714e963]" started-at="2024-04-20T10:17:01 IST" name="tearDown" finished-at="2024-04-20T10:17:02 IST" duration-ms="1455" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.Tc_001_AccountRegistrationTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
